### PR TITLE
Refine README doc of Adaptive-tool

### LINKED ISSUE #2714: Refine adaptive-tools docs
Before shipping adaptive-tools to VSCode marketplace.  We need to refine the readme page. 

https://github.com/microsoft/BotBuilder-Samples/blob/main/experimental/adaptive-tool/README.md

Current issues
* The links to gif is broken, we need to make sure it works after publishing and before publishing (in github)
* Lots of casing issues and typos
* We are missing some general introduction\summary of this extension
* We should also introduce each feature more
* We are also missing the introduction of debugging features (debugging .dialog as least)
* We probably want to move the "how to use" section early

### PR DESCRIPTION
**Pull request by Danieladu**
Fixes #2714

Refine README doc of Adaptive-tool

## Proposed Changes
- [x] Fix Gif link issue
- [x] Casing issue and typo fix
- [x] Add `Quick Start` section
- [x] Add general introduction\summary of this extension
- [x] Add setting/command section
- [x]  Introduce each feature more
- [x] Introduce debugging features more
- [x] move the "how to use" section

### CODE REVIEW

File: experimental/adaptive-tool/README.md [hunk 1]
```diff
@@ -1,57 +1,118 @@
...
-- syntax highlighting
-![lg_syntax_highlighting](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/master/experimental/adaptive-tool/resources/images/lg_syntax_highlighting.png)
+- Template definition
+![template_definition](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/template_definition.gif)
 
-- diagnostic check
-![lg_diagnostic](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/master/experimental/adaptive-tool/resources/images/lg_diagnostic.gif)
+## Debugging Feature
+### Setting up and using Visual Studio Code to run client and server
+#### Setting up
```
Comment (boydc2014) [inline experimental/adaptive-tool/README.md:52]: I think in here, we want to give an E2E setup of debugging .dialog for example, including, checking out which project (maybe testbotJson in dotnet), how to run, how to setup vscode like.

File: experimental/adaptive-tool/README.md [hunk 2]
```diff
@@ -1,57 +1,118 @@
...
+To configure Visual Studio Code you need to add a target in your launch.settings file.
 
-- template reference hover
-![template_hover](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/master/experimental/adaptive-tool/resources/images/template_hover.png)
+* **Bot: Launch language server and client on vscode** - Configuration for building and launching your client on vscode and connecting to server
+Example is:
+```json
```
Comment (boydc2014) [inline experimental/adaptive-tool/README.md:57]: I think those two configuration files is not for debugging .dialog or .lg, right?

File: experimental/adaptive-tool/README.md [hunk 3]
```diff
@@ -1,57 +1,122 @@
 # Bot Framework Adaptive Tool
-BotFramework adaptive tool is a vscode extension to help developers to improve the fficiency of handering LG/LU files.
+Bot Framework Adaptive Tool is a Visual Studio Code extension that helps developers handle LG (.lg) and LU (.lu) files efficiently.
```
Comment (chrimc62) [inline experimental/adaptive-tool/README.md:2]: >LG (.lg [](start = 91, length = 7)

Does it handle .qna as well?  If not, we should extend it to handle .qna as well.

File: experimental/adaptive-tool/README.md [hunk 4]
```diff
@@ -1,57 +1,122 @@
...
+- LG/LU completion
+- LG template and function hover
+- LG template definition
+- LG template, function, and structure suggestions
+- Dialog (.dialog) debugging
+- LG debugging
+
+# Quick Start
+## Install from the marketplace
+1. Install the [adaptive-tool](https://marketplace.visualstudio.com/items?itemName=adaptive-tool) from the [Visual Studio Code marketplace](https://marketplace.visualstudio.com/vscode).
```
Comment (chrimc62) [inline experimental/adaptive-tool/README.md:16]: >https://marketplace.visualstudio.com/items?itemName=adaptive-tool [](start = 31, length = 65)

When is this being published?  Link is 404 right now.

File: experimental/adaptive-tool/README.md [hunk 5]
```diff
@@ -1,57 +1,122 @@
...
+1. Open an LG (.lg) or LU (.lu) file and the adaptive-tool extension will activate.
```
Comment (chrimc62) [inline experimental/adaptive-tool/README.md:17]: >Open an LG (.lg) or LU (.lu) file and the adaptive-tool extension will activate. [](start = 3, length = 80)

Say something about opening .dialog for debugging?

File: experimental/adaptive-tool/README.md [hunk 6]
```diff
@@ -1,57 +1,122 @@
...
+
+# Useful Commands
+|Command|Description|
+|-----|---------------|
+|`lgLiveTest.start`|Start a WebView to test the current LG file|
+
+
+# Adaptive-tool Settings
+|Name|Description|
+|-----|---------------|
```
Comment (chrimc62) [inline experimental/adaptive-tool/README.md:103]: Add an intro on how this lets you ignore or add custom function recognition.

**Review by chrimc62 — CHANGES_REQUESTED** [review]
:clock1:

**Comment by chrimc62** [comment]
Very close and this is a big improvement on the current documentation.  Thanks!

File: experimental/adaptive-tool/README.md (hunk 3 above)
Comment (Danieladu) [reply experimental/adaptive-tool/README.md:2]: Not support currently. But we have plans to support .qna

File: experimental/adaptive-tool/README.md (hunk 4 above)
Comment (Danieladu) [reply experimental/adaptive-tool/README.md:16]: Yes. It is just a placeholder.. Maybe release it in October? @boydc2014

File: experimental/adaptive-tool/README.md (hunk 5 above)
Comment (Danieladu) [reply experimental/adaptive-tool/README.md:17]: the Debugging Feature section contains the steps about how to use the extension to debug the dialog/lg

File: experimental/adaptive-tool/README.md (hunk 6 above)
Comment (Danieladu) [reply experimental/adaptive-tool/README.md:103]: Solved

**Review by chrimc62 — APPROVED** [review]
:shipit:

File: experimental/adaptive-tool/README.md (hunk 4 above)
Comment (boydc2014) [reply experimental/adaptive-tool/README.md:16]: @chrimc62 last time my chat with Scott and ChrisMullins, is we will release this along with the patch release, expect in Octoboer

**Pull request merged**